content/events/2023-new-york-city/registration.md

```diff
@@ -0,0 +1,31 @@
++++
+Title = "Registration"
+Type = "event"
+Description = "Registration for devopsdays New York City 2023"
++++
+
+<div style="width:100%; text-align:left;">
+
+<div id=""></div>
+
+<script src="https://www.eventbrite.com/static/widgets/eb_widgets.js"></script>
+
+<script type="text/javascript">
+    var exampleCallback = function() {
+        console.log('Order complete!');
+    };
+
+    window.EBWidgets.createWidget({
+        // Required
+        widgetType: 'checkout',
+        eventId: '',
+        iframeContainerId: '',
+
+        // Optional
+        iframeContainerHeight: 425,  // Widget height in pixels. Defaults to a minimum of 425px if not provided
+        onOrderComplete: exampleCallback  // Method called when an order has successfully completed
+    });
+</script>
```

Comment:
I'm not sure what the behavior of the iframe will be without the config, but its prob fine as is if we disable links/nav to the page